services/frontend/src/services/generate.ts

```diff
@@ -1,7 +1,16 @@
 import { API_SERVER_URL } from "../constants";
 
-export const generateHttp = (data: string) => {
-  return fetch(`${API_SERVER_URL}/generate/`, {
+export const generateHttp = (data: string, manifest: number) => {
+  let endpoint = `${API_SERVER_URL}/generate/`;
+  if (manifest === 0) {
```

Comment:
Why are you using number literals to indicate the type of the manifest? It'd be better if you could create constants for them. `manifestTypes.DOCKER_COMPOSE`.